packages/semcom-core/docs/modules/specifications/pages/637440573-support-registering-components.adoc

```diff
@@ -0,0 +1,93 @@
+== Technical Specification: Support registering of components
```

Comment:
In AsciiDoc, the page title has to start with a single `=`. The other headers should be bumped a level as well.